api/src/ddoc-extraction.js

```diff
@@ -1,22 +1,29 @@
 const _ = require('underscore');
 const db = require('./db');
+const environment = require('./environment');
 const logger = require('./logger');
-const DDOC_ATTACHMENT_ID = 'ddocs/compiled.json';
+
+const BUNDLED_DDOCS = [
+  { attachmentId: 'ddocs/medic.json', targetDb: db.medic },
+  { attachmentId: 'ddocs/sentinel.json', targetDb: db.sentinel },
+  { attachmentId: 'ddocs/users-meta.json', targetDb: db.medicUsersMeta },
+];
 const SERVICEWORKER_ATTACHMENT_NAME = 'js/service-worker.js';
 const SWMETA_DOC_ID = 'service-worker-meta';
 const SERVER_DDOC_ID = '_design/medic';
 const CLIENT_DDOC_ID = '_design/medic-client';
-const environment = require('./environment');
 
-const getCompiledDdocs = () => db.medic
-  .getAttachment(SERVER_DDOC_ID, DDOC_ATTACHMENT_ID)
-  .then(result => JSON.parse(result.toString()).docs)
-  .catch(err => {
-    if (err.status === 404) {
-      return [];
-    }
-    throw err;
-  });
+const getCompiledDdocs = (attachmentId) => {
+  return db.medic
```

Comment:
I think _this_ is correct to query `db.medic` because all the compiled DDocs are attached to the primary `medic` DDoc in the `medic` db.

However the `extractCompiledDdoc` function should query the `targetDb` - I've updated this now.

Reply:
Cool! Thanks!